letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -110,7 +125,7 @@ su_sudo() {
 
 SUDO_ENV=""
 export CERTBOT_AUTO="$0"
-if [ -n "${LE_AUTO_SUDO:+x}" ]; then
```

Comment:
It is a bug that I didn't catch during my review. To fix the bug, the `:` character needs to be removed. I commented [here](https://github.com/certbot/certbot/pull/1969#discussion_r85807991) saying this should be:

```
if [ -n "${LE_AUTO_SUDO+x}" ]; then
```
This is a [POSIX compliant](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02) way to test if `LE_AUTO_SUDO` string is set to anything, including the empty string. The idea is `LE_AUTO_SUDO` is set to the empty string when nothing special (such as running `su` or `sudo`) needs to be done to invoke Certbot. You can see this is used in the `case` statement following this `if`.

 If busybox doesn't recognize this, we could alternatively come up with an arbitrary string to designate this case.